data/json/monsters.json

```diff
@@ -2795,16 +2906,16 @@
     "diff":15,
     "aggression":100,
     "morale":100,
-    "speed":100,
-    "melee_skill":4,
+    "speed":110,
+    "melee_skill":6,
     "melee_dice":3,
     "melee_dice_sides":6,
-    "melee_cut":1,
-    "dodge":1,
+    "melee_cut":4,
+    "dodge":3,
     "armor_bash":5,
-    "armor_cut":1,
-    "vision_day":15,
-    "vision_night":4,
+    "armor_cut":2,
+    "vision_day":25,
+    "vision_night":15,
```

Comment:
Bad eyesight was supposed to be one of biter's gimmicks. It would be better if you buffed its other stats more instead.